### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
- Footer added. Navbar, container and footer now are 3 different divs, i did this so the footer can take full screen width with no fixed position.

## Screenshot
image

## Trello Card
https://trello.com/c/C3BG5TJL/14-footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,90 +9,97 @@
...
-  <h2 class="page-title">Front-End Tools</h2>
-  <div class="tech-section">
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS logo">
-      <p class="tech-title">ReactJS</p>
-      <p class="tech-sub">Release: 2013</p>
-      <p class="tech-desc">
-        React is declarative, component-based Javascript library for building user interfaces.
-      </p>
+  <div class="container">
```
Comment (mmamaniwolox) [inline index.html]: Use a more descriptive class name

File: index.html [hunk 2]
```diff
@@ -9,90 +9,97 @@
...
-    </div>
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/django_icon.svg" alt="DJango logo">
-      <p class="tech-title">Django</p>
-      <p class="tech-sub">Release: 2005</p>
-      <p class="tech-desc">
-        Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.
-      </p>
-    </div>
+  <div class="footer-container">
```
Comment (mmamaniwolox) [inline index.html]: Use `<footer>` tag instead

File: index.html [hunk 3]
```diff
@@ -9,90 +9,97 @@
...
+    <p class="footer-title">We are game changers</p>
```
Comment (mmamaniwolox) [inline index.html]: This is supposed to be quite a big text isnt it? Why dont you use something like `<h3>` or `<h4>` ?

File: scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  font-size: $text-big;
+  margin-bottom: 10px;
+  text-transform: uppercase;
+}
+
+.footer-container {
+  justify-content: center;
+  align-items: center;
+  background-color: $cerulean;
+  bottom: 0px;
```
Comment (mmamaniwolox) [inline scss/footer.scss]: You shouldn't use bottom /top/left/right with an element that's not `position: absolute`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (rodrigoWolox) [reply index.html]: Like this "<footer class="footer-container">" or just "<footer>" and add the styles to the footer tag in the footer.scss?

Comment (mmamaniwolox) [reply index.html]: Like `<footer class="footer-container">`

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**